crates/gpui/src/color.rs

```diff
@@ -1,4 +1,5 @@
 use anyhow::bail;
+use bytemuck::Zeroable;
```

Comment:
do we even have bytemuck dependency on non-linux?

Reply:
we do now :)